roxie/roxiemem/roxiemem.cpp

```diff
@@ -3950,15 +3995,15 @@ class CChunkingRowManager : public CRowManager
     {
         if (memTraceLevel >= 4)
             logctx.CTXLOG("RoxieMemMgr: attachDataBuff() attaching DataBuff to rowMgr - addr=%p dataBuffs=%u dataBuffPages=%u possibleGoers=%u rowMgr=%p", 
-                    dataBuff, dataBuffs, dataBuffPages, atomic_read(&possibleGoers), this);
+                    dataBuff, dataBuffs, dataBuffPages, possibleGoers.load(), this);
 
         dataBuff->Link();
         DataBuffer *last = NULL;
         bool needCheck;
         {
             CriticalBlock b(activeBufferCS);
             DataBuffer *finger = activeBuffs;
-            while (finger && atomic_read(&possibleGoers))
+            while (finger && possibleGoers.load(std::memory_order_acquire))
```

Comment:
From note below on fewtch_sub, does it matter if others see this ?  If not, could it become relaxed.